test/test_transforms_v2_functional.py

```diff
@@ -1365,3 +1365,28 @@ def test_correctness_uniform_temporal_subsample(device):
 
     out_video = F.uniform_temporal_subsample(video, 8)
     assert out_video.unique().tolist() == [0, 1, 2, 3, 5, 6, 7, 9]
+
+
+# TODO: We can remove this test and related torchvision workaround
+# once we fixed related pytorch issue: https://github.com/pytorch/pytorch/issues/68430
+@make_info_args_kwargs_parametrization(
+    [info for info in KERNEL_INFOS if info.kernel is F.resize_image_tensor],
+    args_kwargs_fn=lambda info: info.reference_inputs_fn(),
+)
+def test_memory_format_consistency_resize_image_tensor(test_id, info, args_kwargs):
+    (input, *other_args), kwargs = args_kwargs.load("cpu")
+
+    output = info.kernel(input.as_subclass(torch.Tensor), *other_args, **kwargs)
+
+    error_msg_fn = parametrized_error_message(input, *other_args, **kwargs)
+    assert input.ndim == 3, error_msg_fn
+    input_stride = input.stride()
+    output_stride = output.stride()
```

Comment:
Would it be simpler to check that `output_mem_format == input_mem_format` ?

Reply:
It is a temporary test before fixing https://github.com/pytorch/pytorch/issues/68430 and merging a PR related to that. Also, here tensors are 3D so there is no direct way to get if a tensor is CL. We have to make the following : 
```
t_aux = t if t.ndim == 4 else t[None]
mf = torch.contiguous_format if t_aux.is_contiguous() else torch.channels_last
```
This is also ugly and bulky

Reply:
fair enough - some comments may help to indicate what exactly we're trying to check with those strides. For example it's worth clarifying what the 2 cases below correspond to (`input_stride[-1] == 1` and  `input_stride[0] == 1`)